inference/server/oasst_inference_server/models/fake_data_factories.py

```diff
@@ -0,0 +1,26 @@
+from oasst_inference_server.models import DbChat, DbMessage, DbUser, DbWorker
+from polyfactory.factories.pydantic_factory import ModelFactory
```

Comment:
I'm curious, how much work do we actually save by depending on `polyfactory` here? Do we purely use it for random title generation?

Reply:
Yes, it basically just populates the models with randomized data of the right data type for us. It has some cool features which might become useful in the future. One big advantage I see is if we add new fields to the models, the initialization will still work without changing the loading.

But I understand this might be too much of a dependency for now. One thing could be to split dev and regular requirements to have less of an impact. Or now that I got a better idea about the model structure, I could probably also dump the randomized data to a JSON and load it without Polyfactory.

Let me know what you prefer.

Reply:
Ok - the dep is quite small so I think we are ok to merge this as-is, and it is indeed a cool feature that this will still work without changing loading when new fields are added